.copier/workflows/main.yaml

```diff
@@ -39,7 +39,7 @@ workflow_configs:
   - source: "repo"
     repo: "mongodb/docs-sample-apps"
     branch: "main"  # optional, defaults to main
-    path: ".copier/workflows.yaml"
+    path: ".copier/workflows/config.yaml"
```

Comment:
FWIW, I find this naming a little confusing. We've got a `workflows` directory (plural) but it contains a singular `config.yaml` - _and_ the naming is generic enough that there's no easy way to distinguish it from other workflows if we were to add multiple configs in the future. And it looks like we're hard-coding a singular path for the source/repo/branch. So it _seems_ this implementation will only support a singular workflow file per branch. 

So I'm wondering if this should be named something like `.copier/workflows/main.yaml` since it's the workflow for the main branch, _or_ if we expect each repo to realistically have only one workflow, omit the `workflows` directory entirely and just make it something like `.copier/config.yaml`?